sql/column.go

```diff
@@ -16,6 +16,9 @@
 
 package sql
 
-type RowMetadata struct {
-	Columns []ColumnMetadata
+// ColumnMetadata SQL column metadata.
+type ColumnMetadata interface {
+	GetName() string
+	GetType() ColumnType
+	IsNullable() bool
```

Comment:
Do you think we can get away with dropping `Get` and `Is` from getter names?

Reply:
Prefixes are dropped at https://github.com/hazelcast/hazelcast-go-client/pull/755/commits/18925866ba8c0cc4fa5ef3e0b86741a4779aaad8